network_wrangler/transit/selection.py

```diff
@@ -305,11 +296,8 @@ def _filter_trips_by_trip(
     Returns:
         pd.DataFrame: Filtered trips
     """
-    _missing = set(select_trip_properties.keys()) - set(trips_df.columns)
-    if _missing:
-        raise TransitSelectionNetworkConsistencyError(
-            f"Route selection properties missing from trips.txt: {_missing}"
```

Comment:
This step should happen in validation (original code does this) rather than in execution.